docs/minio-client-complete-guide.md

```diff
@@ -577,9 +526,29 @@ localdir/new.txt:  10 MB / 10 MB  ┃▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 
 ```
 
+<a name="find"></a>
+### Command `find` - Find files and objects
+``find`` command finds files which match the given set of parameters. It only lists the contents which match the given set of criteria.
+
+```sh
+USAGE:
+  mc find PATH [FLAG...]
+
+FLAGS:
+  --help, -h                       Show help.
+  --exec value                     Spawn an external process for each matching object (see FORMAT)
+  --name value                     Find object names matching wildcard pattern.
+  ...
+  ...
+```
+
+*Example: Find all jpeg images from s3 bucket and copy to minio play test bucket continuously.*
+```sh
+mc find s3/buck --name "*.jpg" --watch --exec "mc cp {} play/test"
```

Comment:
s3/buck -> s3/bucket

Reply:
all the way down from mc rm command section, noticed a few inconsistencies

- section heading title FLAGS and OPTIONS are used variably
- indentation is off in several commands from mc rm downwards for the FLAGS section
- Some other commands use the "Help of * " style for the --help, might be good to change them for consistency